cylc/flow/remote.py

```diff
@@ -276,11 +274,9 @@ def remote_cylc_cmd(
     )
 
 
-def remrun(dry_run=False, forward_x11=False, abort_if=None,
-           set_rel_local=False):
+def remrun(forward_x11=False, abort_if=None, set_rel_local=False):
```

Comment:
I was under the impression remrun was going? Is it needed back for spawn on demand?

Reply:
Actually ignore me, it's being removed in platforms - getting confused with the reviews, sorry!